### PR TITLE
[IMP] Show original orders if user try to create an invoice from ddt …

### PR DESCRIPTION
**Pull request by OpenCode**
…related to invoiced orders

For issue https://github.com/OCA/l10n-italy/issues/220

### CODE REVIEW

File: l10n_it_ddt/models/stock_picking_package_preparation.py [hunk 1]
```diff
@@ -154,6 +154,31 @@ def _compute_display_name(self):
...
+            if picking.sale_id and picking.sale_id.invoice_ids]
+        if invoiced_sale:
+            return {
+                'type': 'ir.actions.act_window',
+                'view_type': 'form',
+                'view_mode': 'tree,form',
+                'res_model': 'sale.order',
+                'target': 'current',
+                'domain': '[("id", "in", {ids})]'.format(ids=invoiced_sale),
+                }
```
Comment (eLBati) [inline l10n_it_ddt/models/stock_picking_package_preparation.py]: Should't this be returned only when invoicing is not based on delivery?

Comment (OpenCode) [inline l10n_it_ddt/models/stock_picking_package_preparation.py]: No. This is returned when an order as an invoice yet. In this way when you start to invoicing from order, you continue to invoicing from order.

**Comment by eLBati** [comment]
ok, thanks :+1:

**Comment by paolovalier** [comment]
:+1:

**Pull request merged**